parts/kuberneteskubelet.service

```diff
@@ -42,15 +42,15 @@ ExecStart=/usr/bin/docker run \
         --pod-manifest-path=/etc/kubernetes/manifests \
         --cluster-dns=${KUBELET_CLUSTER_DNS} \
         --cluster-domain=cluster.local \
-        --register-schedulable=${KUBELET_REGISTER_SCHEDULABLE} \
         --node-labels="${KUBELET_NODE_LABELS}" \
         --cloud-provider=azure \
         --cloud-config=/etc/kubernetes/azure.json \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
         --node-status-update-frequency=${KUBELET_NODE_STATUS_UPDATE_FREQUENCY} \
-        --v=2 ${KUBELET_FEATURE_GATES}
+        --v=2 ${KUBELET_FEATURE_GATES} \
+        ${KUBELET_REGISTER_NODE} ${KUBELET_REGISTER_WITH_TAINTS}
```

Comment:
We can't.

Because, parts/kuberneteskubelet.service is shared by master and agent. For agent, those two flags need to be omitted. It is the same reason, why ${KUBELET_FEATURE_GATES} is written in this way.